### PR TITLE
Add remaining LUCI configs

### PR DESCRIPTION
**Pull request by rryk**

### CODE REVIEW

**Comment by rryk** [comment]
I'd like to add @mi-ac to this review, but for some reason it does not show up in the list of the reviewers. Does he also need to be part of the WebAssembly project to be able to review this?

**Comment by dschuff** [comment]
I don't know about the reviewer list but that's a plausible explanation. It looks like everyone I see in that list is in the org. I'm happy to wait for @mi-ac 's review before landing though.

File: cr-buildbucket.cfg [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    builders {
+      name: "linux"
+      dimensions: "cpu:x86-64"
+      dimensions: "os:Ubuntu-14.04"
+    }
+
+    builders {
+      name: "windows"
+      dimensions: "cpu:x86-64"
+      dimensions: "os:Windows"
```
Comment (mi-ac) [inline cr-buildbucket.cfg]: Maybe specify OS version here, too?

File: cr-buildbucket.cfg [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      build_numbers: YES
+    }
+
+    builders {
+      name: "linux"
+      dimensions: "cpu:x86-64"
+      dimensions: "os:Ubuntu-14.04"
+    }
+
+    builders {
```
Comment (mi-ac) [inline cr-buildbucket.cfg]: Pool dimension is missing. Please add one.

Comment (mi-ac) [reply cr-buildbucket.cfg]: Never mind. Realized again that pool will default to bucket name...

**Pull request merged**